pynapple/core/time_series.py

```diff
@@ -1205,15 +1231,18 @@ def __repr__(self):
                 return tabulate([], headers=headers) + "\n" + bottom
 
     def __getitem__(self, key, *args, **kwargs):
```

Comment:
I think we can simplify the getitem by getting rid of the args and kwargs from the function signature:

1. removing the columns parameter passed to the getitem by `_TsdFrameSliceHelper`, which is overwritten anyways in the `__getitem__` of the TsdFrame.
2. removing the **kwargs passed to `TsdFrame` initialization in the  `__getitem__`  of TsdTensor. I don't see when a non-empty kwarg should passed to the TsdTensor `__getitem__` .

the test passes with this modification.

Reply:
ok, makes sense!

Reply:
done